README.md

```diff
@@ -663,6 +690,22 @@ b = 1
 c = 2
 ```
 
+And defining table out-of-order is valid currenctly but discouraged too:
```

Comment:
I would again delete "valid currenctly but ", see above; "table" should be "tables".